img/l/lexoffice.de.svg

```diff
@@ -0,0 +1 @@
rg/2000/svg" width="420.84" height="61.933"><path d="M597.141 111.875C565.434 32.937 489.605 0 389.531 0 257.758 0 169.484 53.438 169.484 179.004c0 125.57 88.274 179.031 220.047 179.031 144.207 0 213.824-72.738 213.824-198.277v-9.961H282.617v-1.242c0-31.7 32.324-80.188 106.914-80.188 44.129 0 75.203 14.914 90.746 43.508zM490.219 214.434v1.246c0 18.027-22.992 73.972-103.172 73.972-86.399 0-104.43-65.894-104.43-73.972v-1.246h207.602" style="fill:#f47218;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.13333 0 0 -.13333 0 61.933)"/><path d="M1320.9 358.027c131.77 0 220.03-53.453 220.03-179.004C1540.93 53.469 1452.67 0 1320.9 0c-131.78 0-220.05 53.469-220.05 179.023 0 125.551 88.27 179.004 220.05 179.004m0-280.968c67.74 0 106.91 36.671 106.91 101.964 0 65.258-39.17 101.934-106.91 101.934-67.76 0-106.93-36.676-106.93-101.934 0-65.293 39.17-101.964 106.93-101.964M1724.11 272.246V12.422h-109.39v259.824h-69.01v73.359h69.01v12.633c0 89.512 27.35 106.289 129.29 106.289 14.92 0 42.27-.625 68.99-1.234V394.91c-18.64.621-36.68 1.242-55.32 1.242-19.28 0-33.57-3.097-33.57-34.183v-16.364h138.82v12.633c0 89.512 27.35 106.289 129.31 106.289 14.91 0 42.25-.625 68.98-1.234V394.91c-18.65.621-36.66 1.242-55.32 1.242-19.27 0-33.56-3.097-33.56-34.183v-16.364h87.65v-73.359h-87.65V12.422h-109.41v259.824h-138.82" style="fill:#47484b;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.13333 0 0 -.13333 0 61.933)"/><path d="M0 12.422h109.402v451.277H0V12.422" style="fill:#f47218;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.13333 0 0 -.13333 0 61.933)"/><path d="M2108.9 386.621h109.38v77.078H2108.9v-77.078M2108.88 345.605h109.38V12.441h-109.38v333.164M2575.69 222.535c-13.05 44.121-46.61 58.422-90.74 58.422-67.77 0-106.91-36.676-106.91-101.934 0-65.293 39.14-101.964 106.91-101.964 48.48 0 85.15 26.105 90.74 67.754h113.14C2668.95 26.719 2584.39 0 2475.61 0c-125.56 0-210.72 53.469-210.72 179.023 0 125.551 88.28 179.004 220.06 179.004 84.53 0 183.36-25.476 203.88-135.492h-113.14M3150.06 111.875C3118.35 32.937 3042.52 0 2942.44 0c-131.76 0-220.03 53.438-220.03 179.004 0 125.57 88.27 179.031 220.03 179.031 144.21 0 213.83-72.738 213.83-198.277v-9.961h-320.73v-1.242c0-31.7 32.31-80.188 106.9-80.188 44.14 0 75.21 14.914 90.75 43.508zm-106.92 102.559v1.246c0 18.027-23 73.972-103.18 73.972-86.39 0-104.42-65.894-104.42-73.972v-1.246h207.6M851.504 95.156 934.98 11.72h167.93L934.98 178.652l-83.476-83.496" style="fill:#47484b;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.13333 0 0 -.13333 0 61.933)"/><path d="m600.102 345.605 167.933-166.953L600.102 11.688h167.933l83.469 83.496 83.484 83.468-166.953 166.953H600.102" style="fill:#f47218;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.13333 0 0 -.13333 0 61.933)"/><path d="m934.98 178.652 167.94 166.938H934.98l-83.472-83.465 83.472-83.473" style="fill:#47484b;fill-opacity:1;fill-rule:nonzero;stroke:none" transform="matrix(.13333 0 0 -.13333 0 61.933)"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 67 44.5"><path fill="#47484b" d="m33.5 33.4 11.1 11.1H67L44.6 22.3l-11 11"/><path fill="#f47218" d="m0 0 22.4 22.3L0 44.5h22.4l11.1-11.1 11.2-11.1L22.4 0H0"/><path fill="#47484b" d="M44.6 22.3 67 0H44.6l-11 11.1 11 11.2"/></svg>
```

Reply:
Can't see the suggested file, where did you get it from?

Reply:
I took the SVG you added and extracted the "x" from it

Reply:
Smart thinking, thank you!

Reply:
You're welcome!